lms/djangoapps/letstudy_api/views.py

```diff
@@ -0,0 +1,165 @@
+"""
+API end-point for creating a new user without password.
+"""
+import logging
+from uuid import uuid4
```

Comment:
Please, check true sorting the import list. 
Style guide: https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#imports-order

Reply:
Done